tests/client_api.cpp

```diff
@@ -1110,7 +1111,18 @@ TEST(ClientAPI, SendToDevice)
                 sleep();
 
         json body{{"messages",
```

Comment:
This creates the messages json manually, that you want to send. Could you change the sent_to_device endpoint to accept a vector of mtx::events::collections::DeviceEvents, and where the endpoint creates the messages json? You don't need to remove the old one and can even keep it around, so that the new endpoint calls the old one internally, but that would make the API more consistent with the send_message and similar other functions.